test/utility/misc.c

```diff
@@ -224,7 +224,7 @@ testutil_cleanup(TEST_OPTS *opts)
 /*
  * testutil_copy_data --
  *     Copy the data to a backup folder. Usually, the data copy is cleaned up by a call to
- *     testutil_clean_test_artifacts.
+ *     testutil_clean_test_artifacts. Expects dir to be an absolute path.
```

Comment:
What part of this function expects `dir` to be an absolute path? Please let me know if I am wrong but AFAIS neither `testutil_remove` nor `testutil_copy_ext` requires arguments to be absolute paths.


If we do want to keep this restriction:

What do you think about renaming argument to `abs_path` (`dir_abs_path`) or any other self-descripting name?

Maybe it's also worth adding some check/assert that this path is absolute as expected (check that it starts with forward slash?)?

Also applicable to other functions updated in this PR.

Reply:
These `testutil` functions no longer expect to be given an absolute path, but instead get the abs path of the current working directory themselves. `getcwd` function gives us an absolute path so no need to check it is absolute.